samvera-labs/clover-iiif
# 2776 documentation static site

This enhances the static build to include documentation from the README.md file.

### To test
1. Pull down the branch
2. run `npm run build:static`
3. run `npx serve static`

Observe the new static build locally

![image](https://user-images.githubusercontent.com/3020266/160665470-1f92966d-6667-4b4a-8516-a985ac183c21.png)
